langchain4j-dashscope/src/test/java/dev/langchain4j/model/dashscope/QwenStreamingChatModelIT.java

```diff
@@ -40,6 +44,192 @@ public void should_send_non_multimodal_messages_and_receive_response(String mode
         assertThat(response.content().text()).endsWith("That's all!");
     }
 
+    @ParameterizedTest
+    @MethodSource("dev.langchain4j.model.dashscope.QwenTestHelper#functionCallChatModelNameProvider")
```

Comment:
Could you please also check `StreamingChatLanguageModelIT` and `StreamingAiServicesSimpleIT` from https://github.com/langchain4j/langchain4j/pull/1989 and make sure `QwenStreamingChatModel` passes them? You can just copy those tests and run them locally, no need to include them in this PR. Thanks!

Reply:
> Could you please also check `StreamingChatLanguageModelIT` and `StreamingAiServicesSimpleIT` from #1989 and make sure `QwenStreamingChatModel` passes them? You can just copy those tests and run them locally, no need to include them in this PR. Thanks!

Yes, the QwenStreamingChatModel can pass the 2 tests.